dcm4che-tool/dcm4che-tool-findscu/src/main/resources/org/dcm4che3/tool/findscu/messages.properties

```diff
@@ -6,8 +6,9 @@ Query/Retrieve, the Modality Worklist Management, the Unified Worklist and \
 Procedure Step, the Hanging Protocol Query/Retrieve and the Color Palette \
 Query/Retrieve Service Class. findscu only supports query functionality using \
 the C-FIND message. It sends query keys to an Service Class Provider (SCP) \
-and waits for responses. Query keys can be specified in DICOM query file(s) \
-dcmfile_in or by options -m and -r.\n\-\n\
+and waits for responses. Query keys can be specified in DICOM binary file(s) \
+dcmfile_in or xml file(s) dcmfile_in.xml or by options -m and -r. \
+For reference, tool provides sample files with query keys on PATIENT, STUDy, SERIES, IMAGE and MWL levels.\n\-\n\
```

Comment:
typo: "STUDy" -> "STUDY"

"For reference, tool provides ..." -> "For reference, sample files are provided in etc/findscu: ..."

Please also specify the names of these sample files and also that they can be found in the folder `etc/findscu`.

Would be nice to also have an Example with an XML file at the end of the README.md (and the output of  `--help`).